### DISCUSSION TITLE
dp-gen for electrode interface system

### DISCUSSION THREAD

**Discussion by jwchen25** [deepmodeling/dpgen#1335] [Q&A] [answered]
Hello, dear software developers, thank you very much for your time. Can dp-gen be used to generate the configurations of training data for the electrode interface systems (like Au/H2O interface)? In the dp-gen document, bulk and surface are mentioned, but the interface system involves surface and electrolyte molecules.

In addition, can dp-gen be used with JDFTx software?

Thanks a lot for your help!

**Comment by wanghan-iapcm** [comment, marked as answer]
The provided utilities for surface and bulk are used for generating data but not compulsory.  Actually any configuration is allowed to be used in dpgen.  You may need to generate the electrode interface systems by yourself. 
JDFTx is not supported by far.

**Reply by jwchen25** [reply]
Thanks a lot for your time and reply!
